### PR TITLE
[releng] Perform some automatic checks on pull requests

### PR DESCRIPTION
**Pull request by sbegaudeau**
Signed-off-by: Stéphane Bégaudeau <stephane.begaudeau@obeo.fr>

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: Pull Request Checks
```
Comment (lfasani) [inline .github/workflows/pull-request.yml:1]: The existing `ip-verify.yml` use an action instead.
Here: `Check the pull request`

File: scripts/check-changelog.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2021 Obeo.
```
Comment (lfasani) [inline scripts/check-changelog.js:2]: 2022

File: scripts/check-changelog.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+const fs = require('fs');
+
+const workspace = process.env.GITHUB_WORKSPACE;
+const event = process.env.GITHUB_EVENT;
+
+const body = JSON.parse(event);
+const baseSHA = body.pull_request.base.sha;
+const headSHA = body.pull_request.head.sha;
+
+const gitLogCommand = `git log --format=format:%H ${baseSHA}..${headSHA}`;
```
Comment (lfasani) [inline scripts/check-changelog.js:23]: we could add the first line of the commit message in action to the commit id

**Review by lfasani — CHANGES_REQUESTED** [review]

File: scripts/check-changelog.js (hunk 2 above)
Comment (sbegaudeau) [reply scripts/check-changelog.js:2]: Done

File: scripts/check-changelog.js (hunk 3 above)
Comment (sbegaudeau) [reply scripts/check-changelog.js:23]: In this command? Why? That's a git command to compare some commits

**Review by gcoutable — APPROVED** [review]

**Pull request merged**